feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature-1 branch
```

Comment:
Your decision to add this line of code is awsome and really very, very good!

Reply:
Thanks a lot for your comment! I appreciate that very much!